x-pack/plugins/enterprise_search/common/types/connectors.ts

```diff
@@ -38,11 +38,47 @@ export interface IngestPipelineParams {
   run_ml_inference: boolean;
 }
 
+export interface FilteringRule {
+  created_at: string;
+  field: string;
+  id: string;
+  order: number;
+  policy: string;
+  rule: string;
+  updated_at: string;
+  value: string;
+}
+
+export interface FilteringValidation {
+  ids: string[];
+  messages: string[];
+}
+
+export interface FilteringRules {
+  advanced_snippet: {
+    created_at: string;
+    updated_at: string;
+    value: object;
+  };
+  rules: FilteringRule[];
+  validation: {
+    errors: FilteringValidation[];
+    state: string;
```

Comment:
suggestion: We should probably define this as an enum with defined values, rather than a free-for-all string. I'm wondering whether we need to do the same for things like `policy` and `rule`